versioning.md

```diff
@@ -30,6 +28,46 @@ Whether a new version is a patch release, a minor release or a major release sho
 following Semantic Versioning. https://semver.org/
 
 
+## Changelog
+
+All repositories that are versioned using the scheme described on this page should keep a changelog. A file called CHANGELOG.md should be in the root of the repository.
+
+It's format should follow https://keepachangelog.com/en/1.0.0/
+
+To start a new repository that has no releases, the initial changelog should be:
+
+```
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+```
+
+
+## Process for making a new release
+
+These steps depend on each other so carry them out in order.
+
+### Commit to master branch
+
+A special commit should be made to the master branch.
```

Comment:
I have mixed feelings about this.

Reply:
It's maybe unclear what "a special commit" means. Many projects are protected so it would really be a very small pull request. With the approval setting, that would also mean that approval would  be required for a release, which is maybe a good thing.

That could be made clearer.

But were there other alternatives you were thinking about?

Reply:
Pull requests is fine, that's what I was thinking about.